pkgs/by-name/do/dopamine/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchurl,
+  appimageTools,
+}:
+appimageTools.wrapType2 rec {
+  pname = "dopamine";
+  version = "3.0.0-preview.25";
+
+  src = fetchurl {
+    url = "https://github.com/digimezzo/dopamine/releases/download/v3.0.0-preview.25/Dopamine-3.0.0-preview.25.AppImage";
+    hash = "sha256-APvSPIjO3pzD5rBmIeS/6tZIWb0K4/zEn/W1lPqkABs=";
+  };
+
+  extraInstallCommands =
+    let
+      contents = appimageTools.extract { inherit pname version src; };
+    in
+    ''
+      install -Dm644 ${contents}/dopamine.desktop $out/share/applications/dopamine.desktop
+      substituteInPlace $out/share/applications/dopamine.desktop \
+        --replace 'Exec=AppRun' 'Exec=${meta.mainProgram}'
+      cp -r ${contents}/usr/share/icons $out/share
+    '';
+
+  meta = {
+    changelog = "https://github.com/digimezzo/dopamine/blob/${version}/CHANGELOG.md";
+    description = "The audio player that keeps it simple";
+    homepage = "https://github.com/digimezzo/dopamine";
+    license = lib.licenses.gpl3Only;
+    mainProgram = "dopamine";
+    maintainers = with lib.maintainers; [ Guanran928 ];
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = [ "x86_64-linux" ];
```

The Appimage only works in `x86_64-linux`, right?